smartapps/smartthings/ecobee-connect.src/ecobee-connect.groovy

```diff
@@ -201,7 +201,7 @@ def callback() {
 			redirect_uri: callbackUrl
 		]
 
-		def tokenUrl = "https://www.ecobee.com/home/token?${toQueryString(tokenParams)}"
+		def tokenUrl = "https://api.ecobee.com/home/token?${toQueryString(tokenParams)}"
```

Comment:
since this is the same API uri that is used in other places, e.g. L187,  could we use that instead of a hard coded version?
i.e. def tokenUrl = "${apiEndpoint}/home/token?${toQueryString(tokenParams)}"